sources/europe/fi/fta-digiroad-functional.geojson

```diff
@@ -0,0 +1,50 @@
+{
+    "type": "Feature",
+    "properties": {
+        "type": "wms",
+        "name": "FTA Road Network",
+        "id": "fta-digiroad-functional",
+        "country_code": "FI",
+        "description": "Digiroad national road network from the Finnish Transport Agency, functional road classes.",
+        "url": "https://extranet.liikennevirasto.fi/inspirepalvelu/beta/wms?SERVICE=WMS&FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=dr_tielinkki_toim_lk&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+        "icon": "https://www.liikennevirasto.fi/livi-theme/images/general/liikennevirasto_logo_2x.png",
+        "min_zoom": 3,
+        "max_zoom": 20,
+        "overlay": true,
+        "attribution": {
+            "required": true,
+            "text": "\u00a9 Liikennevirasto",
+            "url": "https://www.liikennevirasto.fi/web/en/open-data/digiroad/"
+        },
+        "license_url": "https://lookaside.fbsbx.com/file/Digiroad_permission_23032017.pdf?token=AWzxeAA6eoURtiEfhamunEmttk0mdOG8xdWL0xQJ2ycs6k0AB2M3DQVY11G1Cg8VQpkoW8PoiPfXFrocjWpmietbZi5EtBGPk-coh5sgCxeArDFwVSmmEaCRccgzstqIN4bA5PYnXJSISjW-K8AnAcaw",
```

Comment:
I cannot access this pdf document. Am I doing something wrong?

Reply:
Apologies, here is the correct link: https://www.facebook.com/download/preview/1324879040926767

(You will need to be logged into a Facebook account first)